CorsixTH/Lua/languages/czech.lua

```diff
@@ -3273,151 +3390,155 @@ multiplayer = {
-  ingame_gamespeed_thensome = "A ještě trochu navíc",
-  ingame_quickLoad = "Rychlé načtení",
+  ingame_gamespeed_thensome = "A ještě o trochu víc",
+  ingame_quickLoad = "Rychlé načtení hry",
   ingame_recallPosition_5 = "5",
   global_window_close_alt = "Zavřít okno",
   ingame_storePosition_6 = "6",
   ingame_rotateobject = "Otočit objekt",
-  ingame_poopStrings = "Vypsat řetězce",
-  caption_panels = "Kláv. panelu",
+  ingame_poopStrings = "Vypsat text. řetězce",
+  caption_panels = "Zkratky panelů (výzkum, grafy)",
   ingame_storePosition_3 = "3",
   ingame_gamespeed_normal = "Normální",
-  button_gameSpeedKeys = "Klávesy rychlosti hry",
-  ingame_reset_zoom = "Obnovit přiblížení",
-  panel_storePosKeys = "Uložit klávesy poz. kamery",
+  button_gameSpeedKeys = "Zkratky rychlosti hry",
+  ingame_reset_zoom = "Obnovit výchozí přiblížení",
+  panel_storePosKeys = "Uložit pozici kamery",
+  ingame_panel_machineMenu = "Správa strojů",
+  ingame_toggleTransparent = "Vypnout/zapnout průhlednost stěn",
 }
 update_window = {
-  caption = "Dostupná aktualizace!",
+  caption = "Vyšla nová verze CorsixTH!",
   new_version = "Nová verze:",
-  current_version = "Současná verze:",
-  download = "Přejít na stránku ke stažení",
-  ignore = "Přeskočit a přejít do hlavní nabídky",
+  current_version = "Nainstalovaná verze:",
+  download = "Chci novou verzi",
+  ignore = "Prozatím ignorovat",
 }
 customise_window = {
-  average_contents = "Průměrný obsah",
+  average_contents = "Objekty navíc",
   option_on = "Zapnuto",
   paused = "Stavět při pozastavení",
   option_off = "Vypnuto",
   intro = "Přehrát úvodní video",
-  caption = "Vlastní nastavení",
+  caption = "Další nastavení",
   back = "Zpět",
-  movies = "Globální kontrola videí",
-  volume = "Zkrátka snížení hlasitosti",
-  aliens = "Pacienti s mimozemskou DNA",
-  fractured_bones = "Zlomené kosti",
-  remove_destroyed_rooms = "Odklidit zničenou místnost.",
+  movies = "Videa",
+  volume = "Změna zkratky sez. nemocí",
+  aliens = "Mimozem. DNA jen pohotovost",
+  fractured_bones = "Zak. Pacientky se Zlom. kostmi",
+  remove_destroyed_rooms = "Odstranění zničené místnosti",
 }
 custom_campaign_window = {
   caption = "Vlastní kampaň",
   start_selected_campaign = "Spustit kampaň",
+  duplicates_warning = "%d kampaní se stejnými jmény bylo skryto (nesmějí mít stejná jména).",
 }
 warnings = {
-  levelfile_variable_is_deprecated = "Upozornění: Úroveň '%s' ve svém souboru obsahuje zastaralé určení proměnné.'%LevelFile' bylo přejmenováno na '%MapFile'. Pro aktualizaci úrovně prosím použijte editor map.",
-  newersave = "Pozor! Nahráli jste uloženou hru z novější verze. Nedoporučujeme pokračovat z důvodu možných pádů hry. Na vlastní riziko.",
+  levelfile_variable_is_deprecated = "Upozornění: Úroveň '%s' obsahuje starý formát definice nastavení." ..
+                                     "Proměnná '%LevelFile' byla změněna na '%MapFile'. Prosím, kontaktujte autora této úrovně.",
```

Comment:
Concatenating two strings doesn't add a space (think you just missed this one). You'll probably want one at the end of 3539 or start of 3540.

Reply:
It can merge without change, of course.